src/applications/edu-benefits/1990/helpers.jsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import { transformForSubmit } from 'us-forms-system/lib/js/helpers';
 
+export function prefillTransformer(pages, formData, metadata) {
+
+  let newContactInfo = {};
+
+  if (formData && formData.email) {
+    newContactInfo = {
+      email: formData.email,
+      'view:confirmEmail': formData.email
+    };
+  }
+
+  if (formData && formData.homePhone) {
+    newContactInfo = {
+      ...newContactInfo,
```

Comment:
You could set `homePhone` on `newContactInfo` directly. You created it in this function, so mutating it won't cause any issues.